code/game/objects/items/stacks/nanopaste.dm

```diff
@@ -28,14 +30,17 @@
 		var/mob/living/carbon/human/H = M
 		var/obj/item/organ/external/S = H.get_organ(user.zone_sel.selecting)
 
-		if(S.open >= 2)
-			if (S && (S.robotic >= ORGAN_ROBOT))
-				if(!S.get_damage())
-					user << "<span class='notice'>Nothing to fix here.</span>"
-				else if(can_use(1))
-					user.setClickCooldown(DEFAULT_ATTACK_COOLDOWN)
-					S.heal_damage(15, 15, robo_repair = 1)
-					H.updatehealth()
-					use(1)
-					user.visible_message("<span class='notice'>\The [user] applies some nanite paste on [user != M ? "[M]'s [S.name]" : "[S]"] with [src].</span>",\
-					"<span class='notice'>You apply some nanite paste on [user == M ? "your" : "[M]'s"] [S.name].</span>")
+		if (S && (S.robotic >= ORGAN_ROBOT))
+			if(!S.get_damage())
+				user << "<span class='notice'>Nothing to fix here.</span>"
+			else if(can_use(1))
+				user.setClickCooldown(DEFAULT_ATTACK_COOLDOWN)
+				if(S.open >= 2)
+					if(do_after(user,5 * toolspeed))
+						S.heal_damage(15, 15, robo_repair = 1)
```

Comment:
Bump this to 20~25, actually. If you've opened them up, might as well be better than the normal surgeries, yeah?